editor/core/workspace.js

```
@@ -0,0 +1,171 @@
+  if (!shadow?.outputConnection) return;
+  try {
+    connection.connect(shadow.outputConnection);
+  } catch (error) {
+    shadow.dispose(false, true);
+  }
+};
+
+const ensureLiteralShadowsForBlock = (block) => {
+  if (!block || block.isShadow?.() || block.isInsertionMarker?.()) return;
+  if (block.workspace?.isFlyout) return;
+  block.inputList?.forEach((input) => ensureLiteralShadowForInput(block, input));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)